### PR TITLE
Add webportal.rst and webportal_zh.rst in experiment folder

### PR DESCRIPTION
**Pull request by Lijiaoa**
### Description ###
- put webui doc in `experiment` directory 
- put tensorboard doc in webui
- update webui doc content
- rename webui to webportal

image

### Checklist ###
  - [x] Chinese doc
  - [x] Add tensorboard content

### CODE REVIEW

File: docs/source/experiment/WebUI.rst [hunk 1]
```diff
@@ -0,0 +1,375 @@
+WebUI
+#####
```
Comment (ultmaster) [inline docs/source/experiment/WebUI.rst:2]: The top level should be underlined with `======`.
Second level is `------`.
Third level is `^^^^^^`.
Fourth level is `""""""`.

File: docs/source/experiment/WebUI.rst [hunk 2]
```diff
@@ -0,0 +1,375 @@
...
+
+
+Q&A
+***
+
+There are many trials in the detail table but ``Default Metric`` graph is empty.
```
Comment (ultmaster) [inline docs/source/experiment/WebUI.rst:8]: graph -> chart

File: docs/source/experiment/WebUI.rst [hunk 3]
```diff
@@ -0,0 +1,375 @@
...
+
+* Download the experiment results(``experiment config``, ``trial message`` and ``intermeidate metrics``) from ``Experiment summary`` and then upload these results in your issue.
+
+
+.. image:: ../../img/webui-img/summary.png
+   :target: ../../img/webui-img/summary.png
+   :alt: summary
+
+
+What should you do when your experiment is error
```
Comment (ultmaster) [inline docs/source/experiment/WebUI.rst:25]: has errors

File: docs/source/experiment/WebUI.rst [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+* And then click the ``learn about`` to download ``nni-manager`` and ``dispatcher`` logfile.
+* Please file an issue from the `Feedback` in the `About` and upload above message.
+
+
+.. image:: ../../img/webui-img/experimentError.png
+   :target: ../../img/webui-img/experimentError.png
+   :alt: experimentError
+
+
+What should you do when your trial is failed
```
Comment (ultmaster) [inline docs/source/experiment/WebUI.rst:39]: fails

File: docs/source/experiment/WebUI.rst [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+   :alt: customizedTrial
+
+
+
+* ``Log model`` will help you find the error reason. There are three buttons ``View trial log``, ``View trial error`` and ``View trial stdout`` on local mode. If you run on the OpenPAI or Kubeflow platform, you could see trial stdout and nfs log.
+  If you have any question you could tell us in the issue.
+
+  **local mode:**
+
+.. image:: ../../img/webui-img/detail/log-local.png
```
Comment (ultmaster) [inline docs/source/experiment/WebUI.rst:61]: This should have the same indent as ``local mode:``.

File: docs/source/experiment/WebUI.rst (hunk 1 above)
Comment (Lijiaoa) [reply docs/source/experiment/WebUI.rst]: fixed

File: docs/source/experiment/WebUI.rst (hunk 2 above)
Comment (Lijiaoa) [reply docs/source/experiment/WebUI.rst]: fixed

File: docs/source/experiment/WebUI.rst (hunk 3 above)
Comment (Lijiaoa) [reply docs/source/experiment/WebUI.rst]: had updated

File: docs/source/experiment/WebUI.rst (hunk 4 above)
Comment (Lijiaoa) [reply docs/source/experiment/WebUI.rst]: update

File: docs/source/experiment/WebUI.rst (hunk 5 above)
Comment (Lijiaoa) [reply docs/source/experiment/WebUI.rst]: fixed

File: docs/source/experiment/webui.rst [hunk 6]
```diff
@@ -0,0 +1,413 @@
+WebUI
+=====
+
+
+Q&A
+---
+
+There are many trials in the detail table but ``Default Metric`` chart is empty.
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (J-shang) [inline docs/source/experiment/webui.rst:9]: remove `.` or add `?` at the end of each question?

File: docs/source/experiment/webui.rst [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+
+`The discussion <https://github.com/microsoft/nni/discussions/4289>`_ could help you.
+
+
+
+Experiments management
+----------------------
+
+Experiments management page could manage many experiments on your machine. 
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Comment (J-shang) [inline docs/source/experiment/webui.rst:96]: Is this line a title?

File: docs/source/experiment/webui.rst [hunk 8]
```diff
@@ -0,0 +1,413 @@
...
+
+
+.. image:: ../../img/webui-img/managerExperimentList/expFilter.png
+   :target: ../../img/webui-img/managerExperimentList/expFilter.png
+   :alt: filter button
+
+
+
+View summary page
+-----------------
```
Comment (J-shang) [inline docs/source/experiment/webui.rst:137]: seems this title level should align with `View job default metric`, `View ...`

File: docs/source/experiment/webui.rst (hunk 6 above)
Comment (Lijiaoa) [reply docs/source/experiment/webui.rst:9]: ok, remove `.`

File: docs/source/experiment/webui.rst (hunk 7 above)
Comment (Lijiaoa) [reply docs/source/experiment/webui.rst:96]: will change it to paragraph

**Review by J-shang — APPROVED** [review]

**Pull request merged**